languages/ruby/spec/oso/polar/data_filtering/gitclub.polar

```diff
@@ -36,15 +36,13 @@ resource Issue {
   "edit" if "writer" on "parent";
 }
 
-has_role(user: User, name: String, org: Org) if
-  role in user.org_roles and
-  role matches { name: name, org: org };
+has_role(_: User{org_roles}, name: String, org: Org) if
+  r in org_roles and r matches { name, org };
 
-has_role(user: User, name: String, repo: Repo) if
-  role in user.repo_roles and
-  role matches { name: name, repo: repo };
+has_role(_: User{repo_roles}, name: String, repo: Repo) if
```

Comment:
I don't think these changes (to compress this down) are really needed. But if you want to keep them, does this file exist in other test suites? All should match.

Reply:
Nah whatever, this was from when I actually had two policy files & was confused about why a test was failing (it was reading the wrong file). I'll revert it (although personally I prefer this style of rules)